### ISSUE TITLE
🔥The new elementor pro 3.6 update interferes with wp-rocket.

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#17702]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

hello

After the new update, if we enable this option, parts of the site will be corrupted. https://prnt.sc/26nzphh
For example, the slider navigation points are not displayed on the mobile phone. On the other hand, if we do not enable this option, other problems will arise.
Another problem with activating this option is that sticky sections fail. For example I have a navigation bottom in the footer section of my website which is sticky at the bottom. (On mobile) If this option is enabled, the navigation bottom is no longer sticky.

Please check this issue. it's too important.

### Steps to reproduce

Install the rocket plugin next to elementor and elementor pro. Activate the opposite option and check the above (slider points and sticky section).

**The very important point is that the problem of sticky sections in elementor editor works properly. But it does not work on the website.**

Another interference is that if wp-rocket is enabled, the pop-up display conditions will not work.

thank you

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

elementor Version 3.5.5
elementor pro Version 3.6.0
wp-rocket Version 3.10.7
wordpress Version 5.9
theme : hello elementor Version 2.5.0

**Comment by Michaelgimii** [comment]
I have this problem

**Comment by black-eye** [comment]
@79ho3ein, I'd recommend to create the same report on WP Rocket's Github (https://github.com/wp-media/wp-rocket/issues) as well. I think they will be more likely in the mood to solve the problem (if it's solvable on their side). I can imagine Elementor's response would take while (and would probably be that's the other's plugin problem anyway).

**Comment by 79ho3ein** [comment]
@black-eye , Everything worked fine before version 3.6. But after the elementor pro update, these problems appeared. I do not know, maybe I will stay in 3.5.2 for now ...

**Comment by black-eye** [comment]
@79ho3ein Yes, I get your point, on the other hand I have a decent record of trying to report the bugs to Elementor (and some to WP Rocket as well) and if I had to bet, who will resolve the problem first, I'd bet on the Rocket team. ;-) They are much more responsive (and responsible), at least from mine experience.

Not to mention, that in this case there could be a better chance to find the solution on their side. Yes, the problem was introduced by Elementor's update, something has change there, apparently, but it's Rocket's feature, which is responsible for combiniing the scripts. There might be some exceptions for the former Elementor version, which aren't met now or something like this, who knows. Long story short, it's better to have two irons in the fire... ;-)

**Comment by 79ho3ein** [comment]
@black-eye , You are definitely right. Thank you. Do you experience this problem too?

**Comment by black-eye** [comment]
@79ho3ein 
>Do you experience this problem too?

I haven't updated to 3.6 yet, so no. :-) I'm in no rush in updating Elementor in general, and in this version there's nothing I would be craving for. But I keep an eye on the possible problems and this one meets my usual set up.

**Comment by 79ho3ein** [comment]
thank you @black-eye

**Comment by 79ho3ein** [comment]
@black-eye , Yes, I found a solution. With the recommendation of wp-rocket, the two options of combining JavaScript and combining css, if http2 is enabled, it is better not to be enabled. I disabled these two options and the problem was solved. Of course, the problem with the element was the option to combine JavaScript. **But there are two issues here and I do not know which one is right.**
Websites that support http2 combine css and js requests by default. So we do not need to enable these two options in rocket.
**When I enable this option, the number of website requests reaches 66. But when I disable both (according to Rocket recommendation), the number of requests reaches 145, but the speed is great.**
So I'm confused which one is right.
